Do not remove MacPorts prefix from PATH because we might want to compile
with ccache or distcc.
--- configure.orig	2020-07-18 17:49:15.000000000 -0500
+++ configure	2021-03-08 04:12:32.000000000 -0600
@@ -2511,34 +2511,6 @@
 $as_echo "$MACPORTS_VERSION" >&6; }
 
 
-# Check user PATH for MP paths, and remove as necessary.
-
-	oldprefix=$prefix
-	if test "x$prefix" = "xNONE" ; then
-		prefix=$ac_default_prefix
-	fi
-	oldPATH=$PATH
-	newPATH=
-	as_save_IFS=$IFS; IFS=$PATH_SEPARATOR
-	for as_dir in $oldPATH
-	do
-		IFS=$as_save_IFS
-		case "$as_dir" in
-			$prefix/*)
-				;;
-			*)
-				if test -z "$newPATH"; then
-					newPATH=$as_dir
-				else
-					newPATH=$newPATH$PATH_SEPARATOR$as_dir
-				fi
-				;;
-		esac
-	done
-	PATH=$newPATH; export PATH
-	PATH_CLEANED=$newPATH
-
-	prefix=$oldprefix
 
 
 	# Check whether --enable-flag-sanitization was given.
